### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (span[itemprop="name"])

### ISSUE THREAD

**Issue by brianteeman** [joomla/cassiopeia#113]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI))
#### Target application
[Home](http://localhost/cass/)
#### Element path
span[itemprop="name"]
#### Snippet
`<span itemprop="name">Home</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.37 (foreground color: #6d757e, background color: #f7f7f7, font size: 14.3pt (19.072px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 85.0.4183.102
This accessibility issue was found using Accessibility Insights for Web 2.21.1 (axe-core 4.0.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by richard67** [comment]
Should be solved with PR #152 . I've just visually tested.
